content/kubermatic/master/installation/add_seed_cluster/_index.en.md

```diff
@@ -95,30 +48,26 @@ It's also advisable to install the `s3-exporter` Helm chart, as it provides basi
 With this you can install the chart:
 
 ```bash
-cd kubermatic-installer
 helm --tiller-namespace kubermatic upgrade --install --values /path/to/your/helm-values.yaml --namespace nodeport-proxy nodeport-proxy charts/nodeport-proxy/
 helm --tiller-namespace kubermatic upgrade --install --values /path/to/your/helm-values.yaml --namespace minio minio charts/minio/
 helm --tiller-namespace kubermatic upgrade --install --values /path/to/your/helm-values.yaml --namespace s3-exporter s3-exporter charts/s3-exporter/
 ```
 
-### 3. Create Seed Resource
 
-To connect the new seed cluster with the master, you need to create a kubeconfig Secret and a Seed resource
-**in the master cluster**.
 
-{{% notice warning %}}
-Do not install the `kubermatic-operator` chart into seed clusters. It's possible to run master and seed in the same
-Kubernetes cluster, but this still means only a single operator is deployed into the shared cluster.
-{{% /notice %}}
+
+### Add the Seed Resource
+
+To connect the new seed cluster with the master, you need to create a kubeconfig Secret and a Seed resource.
+
+You will add the **master cluster** as **seed cluster**-
 
 Make sure the kubeconfig contains static, long-lived credentials. Some cloud providers use custom authentication providers
 (like GKE using `gcloud` and EKS using `aws-iam-authenticator`). Those will not work in Kubermatic’s usecase because the
-required tools are not installed inside the cluster environment. You can use the `kubeconfig-serviceaccounts.sh` script from
-the kubermatic-installer repository to automatically create proper service accounts inside the seed cluster with static
-credentials:
+required tools are not installed inside the cluster environment. You can use the `kubeconfig-serviceaccounts.sh` script to 
+automatically create proper service accounts inside the seed cluster with static credentials:
```

Comment:
Is this script already in the kubermatic repo? Otherwise people will not find it.